hypothesis-python/tests/cover/test_targeting.py

```diff
@@ -78,9 +78,8 @@ def everything_except(type_):
 @example(float("nan"), "")
 @example(float("inf"), "")
 @example(float("-inf"), "")
-@example("1", "Non-float observations are invalid")
```

Comment:
We can leave this line in, perhaps relabelled `"Non-numeric ..."`, because string values are still invalid.